Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -137,6 +145,10 @@ public extension ProtobufFloat {
         }
         return true
     }
+
+    public static func encodedSize(of value: Float) -> Int {
+        return MemoryLayout<Float>.size
```

Comment:
Good!  I found `MemoryLayout<>` just recently and have been meaning to go through and adopt it.